easybuild/easyblocks/generic/configuremake.py

```diff
@@ -79,6 +81,8 @@ def configure_step(self, cmd_prefix=''):
             for (key, val) in tar_vars.items():
                 self.cfg.update('preconfigopts', "%s='%s'" % (key, val))
 
+         # Ensure that configure has the excutable bit set, needed if a patch created configure itself
+        adjust_permissions('configure', stat.S_IXUSR, add=True)
```

Comment:
this needs a check like:

```python
if os.path.exists('configure'):
```

it's possible that the `configure` script is not present yet, and gets generated 'on-the-fly' by using

```python
preconfigopts = './autogen.sh && '
```